app/frontend/modules/filter_toggle.js

```diff
@@ -0,0 +1,112 @@
+export class FilterToggle {
```

Comment:
Not sure where best to save this file, as it’s not a controller, not is it really something from a vendor. So created a new directory called `modules`.

Reply:
I agree, makes much more sense 😄  Thanks!